utils/db_utils.py

```diff
@@ -98,6 +100,31 @@ def query_confirmed_trips(start_date, end_date):
     # logging.debug("After filtering, the actual data is %s" % df.head().trip_start_time_str)
     return df
 
+def query_demographics(start_date, end_date):
+    start_ts, end_ts = None, datetime.max.timestamp()
+    if start_date is not None:
+        start_ts = datetime.combine(start_date, datetime.min.time()).timestamp()
+
+    if end_date is not None:
+        end_ts = datetime.combine(end_date, datetime.max.time()).timestamp()
```

Comment:
demographic data is not the same as trip data - it is typically only answered once, at the beginning of the program. 
Trips come in throughout the program. We cannot blindly apply the trip filters to the demographic data